package.json

```diff
@@ -3,9 +3,14 @@
   "engines": {
     "node": "12.x || 14.x"
   },
-  "workspaces": [
-    "packages/react"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/react"
+    ],
+    "nohoist": [
+      "**"
```

Comment:
TL:DL; @storybook/addon-docs does hot hoist properly in lerna and gets stuck in the packages/react/node_modules which breaks everything. This prevents the hoisting and fixes the issue.

However, this is like using a sledgehammer on a finishing nail, but it was a seemingly never-ending cycle of adding individual packages to nohoist and running tests again. Things continued to fail in weird ways at various points in either webpack compilation or testing.